cli/flags.rs

```diff
@@ -136,11 +137,25 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
     ).subcommand(
       SubCommand::with_name("xeval")
         .setting(AppSettings::DisableVersion)
-        .about("xargs-like eval script")
-        .arg(
-          Arg::with_name("var")
-            .long("var")
-            .short("V")
+        .about("`xargs`-like eval script")
+        .long_about(
+          "
+`xargs`-like eval script.
+
+Read from standard input and eval code on each whitespace-delimited
+string chunks.
+
+-I/--replvar optionally set variable name for input to be used in eval.
+Otherwise `$` will be used as default variable name.
+
+  cat list.txt | deno xeval 'console.log($)'
+  cat list.txt | deno xeval -I 'val' 'console.log(val)'
+",
+        ).arg(
+          Arg::with_name("replvar")
+            .long("replvar")
```

Comment:
Is long necessary?

Reply:
It is, otherwise this big message would show up in `deno -h`. With `long_about` it will show up in `deno help xeval`

Reply:
I think it is necessary if you want to support long names, e.g. `--replvar`

Reply:
@kevinkassimo you're right, sorry for noise